### PR TITLE
normalization

### PR DESCRIPTION
**Pull request by dvd42**

### CODE REVIEW

File: test/test_image.py [hunk 1]
```diff
@@ -48,3 +48,50 @@ def test_gradcheck(self):
...
+        data = torch.ones(2, 3, 1, 1)
+        data += 2
+        mean = torch.tensor([0.5, 1.0, 2.0])
+        std = torch.tensor([2., 2., 2.])
+
+        # expected output
+        expected = torch.tensor([1.25, 1., 0.5]).repeat(2, 1, 1).view_as(data)
+
+        f = image.Normalize(mean, std)
+        assert_allclose(f(data), expected)
```
Comment (edgarriba) [inline test/test_image.py]: I would add a couple of test more with small image patches (2x2 or 4x4, up to you) to verify that works with 2d arrays and verify the following:
- batch 1 and 1mean/std
- batch N and 1 mean/std
- batch N and N mean/std

File: torchgeometry/image/normalization.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    """Normalize a tensor image or a batch of tensor images
+    with mean and standard deviation. Input must be a tensor of shape (C, H, W)
+    or a batch of tensors (*, C, H, W).
+    Given mean: ``(M1,...,Mn)`` and std: ``(S1,..,Sn)`` for ``n`` channels,
+    this transform will normalize each channel of the input ``torch.*Tensor``
+    i.e. ``input[channel] = (input[channel] - mean[channel]) / std[channel]``
+
+    Args:
+        mean (Tensor): Mean for each channel.
```
Comment (edgarriba) [inline torchgeometry/image/normalization.py]: ```torch.Tensor``` otherwise the docs won't link correctly
